File: C#/SchroederEncryption/messageSecurityTester.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SchroederEncryption
{
    public class messageSecurityTester
    {
        public static void Main(string[] args)
        {

            Console.WriteLine("Please enter a word to be encrypted");
            string user = Console.ReadLine();

            messageSecurity message = new messageSecurity(user);

            Console.WriteLine();

            Console.WriteLine("Encrypted Message: ");

            string encrypted = message.encrypt();

            Console.WriteLine(encrypted);
            Console.WriteLine();
            Console.WriteLine();

            Console.WriteLine("Decrypted Message: ");

            string decrypted = message.decrypt();

            Console.WriteLine(decrypted);

            Console.WriteLine();

        }

    }
}

File: Python/messageSecurityTester.py
from messageSecurity import messageSecurity

userInput = input('Please enter a word to be encrypted: ')
userPasscode = input('Please enter a passcode to validate encryption and decryption: ')
userKeyPath = input('Please enter the path where your key file is stored (default "keys/key.txt")')
userWantNewKey = input('Would you like to generate a new key to encrypt and decrypt this message? (enter "1" for True or "0" for False): ')

message = messageSecurity(userInput, userPasscode, userKeyPath, userWantNewKey)

print('\nEncrypted Message: ')
print(message.encrypt())

print('\n\nDecrypted Message: ')
print(message.decrypt())